apps/s_time.c

```diff
@@ -433,6 +380,13 @@ static SSL *doConnection(SSL *scon, const char *host, SSL_CTX *ctx)
         return NULL;
     }
 
+#if defined(SOL_SOCKET) && defined(SO_LINGER)
+    no_linger.l_onoff  = 1;
+    no_linger.l_linger = 0;
+    (void) setsockopt(SSL_get_fd(serverCon), SOL_SOCKET, SO_LINGER,
+                      &no_linger, sizeof(no_linger));
```

Comment:
Formally speaking it might be more appropriate to implement it as BIO_CTRL. I mean whole point with BIO is to arrange implementation-neutral interface, and this kind of very specific. On the other hand this is special case...